### DISCUSSION TITLE
Code files are not listed on search for specific language

### DISCUSSION THREAD

**Discussion by joshiayush** [github-linguist/linguist#6243] [Q&A] [answered]
Since I created this repo [ai](https://github.com/joshiayush/ai) I couldn't find any code files on search after clicking on the language bar, you can also look at the search results by yourself at https://github.com/joshiayush/ai/search?l=jupyter-notebook. Please help me with this.

Many people say that it's a GitHub problem and check the System status of GitHub I did that and found that all systems are operational but still there is a problem in my repository.

Since I created that repository GitHub is persistent with its weird behaviour. Please tell me what should I do to fix it.

**Comment by lildude** [comment, marked as answer]
> Since I created this repo [ai](https://github.com/joshiayush/ai) I couldn't find any code files on search after clicking on the language bar, you can also look at the search results by yourself at https://github.com/joshiayush/ai/search?l=jupyter-notebook. Please help me with this.

As mentioned in your previous discussion, GitHub's search is completely independent of Linguist. You'll need to contact GitHub support using the "Contact GitHub" link at the bottom of the page and someone can look into this for you. As an aside, using the [new beta search functionality](https://github.blog/2021-12-08-improving-github-code-search/) returns results.
